src/ResourceManager/RedisCache/Commands.RedisCache/Commands/NewAzureRedisCache.cs

```diff
@@ -41,10 +41,10 @@ public class NewAzureRedisCache : RedisCacheCmdletBase
         [ValidateNotNullOrEmpty]
         public string Location { get; set; }
 
-        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "Size of redis cache. Valid values: P1,P2, P3, P4, C0, C1, C2, C3, C4, C5, C6, 250MB, 1GB, 2.5GB, 6GB, 13GB, 26GB, 53GB")]
-        [ValidateSet(SizeConverter.P1String, SizeConverter.P2String, SizeConverter.P3String, SizeConverter.P4String,
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "Size of redis cache. Valid values: P1,P2, P3, P4, P5, C0, C1, C2, C3, C4, C5, C6, 250MB, 1GB, 2.5GB, 6GB, 13GB, 26GB, 53GB, 120GB")]
+        [PSArgumentCompleter(SizeConverter.P1String, SizeConverter.P2String, SizeConverter.P3String, SizeConverter.P4String, SizeConverter.P5String,
```

Comment:
@SiddharthChatrolaMs any test functionality for the new values to ensure that they work on the server?

Reply:
We wanted to make this parameter string so whenever we have new Sku, customers can create new caches without PowerShell update. Existing tests would be enough for that. As we don't test all of the values provided here